app/src/lib/stores/app-store.ts

```diff
@@ -1607,6 +1608,41 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
       return { workingDirectory, selectedFileIDs, diff }
     })
+
+    const inConflictedMerge = status.workingDirectory.files.some(f => {
+      return (
+        f.status === AppFileStatus.Conflicted ||
+        f.status === AppFileStatus.Resolved
+      )
+    })
+
+    // if any files are conflicted or resolved,
+    // we trigger the conflict resolution flow
+    // (if we're not already in it)
+    if (
+      status.currentBranch !== undefined &&
+      inConflictedMerge &&
+      (this.currentPopup === null ||
+        (this.currentPopup !== null &&
+          this.currentPopup.type !== PopupType.MergeConflicts &&
+          this.currentPopup.type !== PopupType.AbortMerge))
+    ) {
+      const possibleTheirsBranches = await getBranchesPointedAt(
+        repository,
+        'MERGE_HEAD'
```

Comment:
Is there a chance that `MERGE_HEAD` might not exist, and we'd need to handle this exit code reported from Git (this is on a repository not addressing merge conflicts):

```
$ git branch --points-at=MERGE_HEAD
error: malformed object name 'MERGE_HEAD'
$ echo $?
129
```

Reply:
it would be hard, to get in that state, but if we do its now handled (7a8d96a)

Reply:
🤔 I was thinking of a different approach here,  unless you care about the error and want to log it...

 - handle the exit code inside  `getBranchesPointedAt ` for a malformed object name provided by `commitish` , using `successExitCodes`
 - return an empty array of branches, rather than letting the error bubble up to the caller

The benefits of guarding in the Git function itself is that other callers who may provide an invalid `committish` won't have to handle the error.

Reply:
@shiftkey would prefer that `getBranchesPointedAt` returned `null` when it has received a malformed `committish`? as the caller, i would like to know whether there are no branches or if the ref is bad. (i think if somehow `MERGE_HEAD` doesn't exist, we don't want to show the popup)

Reply:
You could return `null` in that case